configure.sh

```diff
@@ -16,6 +16,7 @@ Configures the CMAKE build environment.
 --with-msat             build with MathSAT which has a custom non-BSD compliant license.  (default : off)
                         Required for interpolant based model checking
 --with-coreir           build the CoreIR frontend (default: off)
+--with-coreir-extern    build the CoreIR frontend using an externally installed library (e.g. pycoreir) (default: off)
```

Comment:
```suggestion
--with-coreir-extern    build the CoreIR frontend using a globally installed library (e.g. pycoreir) (default: off)
```

I don't feel strongly about this, but thought this wording might be clearer if it's still accurate.